sleap/gui/widgets/video.py

```diff
@@ -970,8 +973,15 @@ def selectInstance(self, select: Union[Instance, int]):
         Returns:
             None
         """
+        
         for idx, instance in enumerate(self.all_instances):
-            instance.selected = select == idx or select == instance.instance
+            if isinstance(select, int):
+                instance.selected = select == idx
+            elif isinstance(select, Instance):
+                instance.selected = select == instance.instance
+            elif isinstance(select, QtInstance.instance):
```

Comment:
We cannot access an instance attribute on a class. If you have some time, I would suggest reading this [article on instance attributes](https://realpython.com/python-classes/#instance-attributes).
```suggestion
            elif isinstance(select, QtInstance):
```